xml/admin_rbd.xml

```diff
@@ -1030,17 +1027,55 @@ EOF</screen>
 <screen>&prompt.cephuser;rbd --cluster remote mirror pool peer bootstrap import \
  --site-name remote image-pool token</screen>
    </sect3>
-   <sect3>
-    <title>Add Cluster Peer</title>
+   <sect3 xml:id="ceph-rbd-mirror-add-peer">
+    <title>Add Cluster Peer Manually</title>
     <para>
-     In order for the <systemitem>rbd-mirror</systemitem> daemon to discover
-     its peer cluster, the peer needs to be registered to the pool. To add a
-     mirroring peer cluster, specify the <command>mirror pool peer
-     add</command> subcommand, the pool name, and a cluster specification:
+     Alternatively to bootstrapping peers as described in <xref
+     linkend="ceph-rbd-mirror-bootstrap-peer"/>, you can specify peers
+     manually. The remote &rbdmirror; daemon will need access to the local
+     cluster to perform mirroring. Create a new local &ceph; user that the
+     remote &rbdmirror; daemon will use, for example
+     <literal>rbd-mirror-peer</literal>:
     </para>
 <screen>
-&prompt.cephuser;rbd --cluster local mirror pool peer add <replaceable>POOL_NAME</replaceable> client.rbd-mirror-peer@remote
-&prompt.cephuser;rbd --cluster remote mirror pool peer add <replaceable>POOL_NAME</replaceable> client.rbd-mirror-peer@local
+&prompt.cephuser;ceph auth get-or-create client.rbd-mirror-peer \
+ mon 'profile rbd' osd 'profile rbd'
+</screen>
+    <para>
+     Store the resulting keyring to the <literal>config-key</literal> store:
+    </para>
+<screen>&prompt.cephuser;ceph config-key rbd-mirror/keyring <replaceable>KEYRING</replaceable></screen>
```

Comment:
Why do you need this? The command below (`rbd ... mirror pool peer add ... --remote-key-file remote-key-file`) will properly store the keyring in the config-key store.

Reply:
I think this may be some confusion. Because the way I read this step vs. the one below is that this step _stores_ the generated key in the config-key store. Whereas the step below provides the rbd-mirror daemon with access to the peer clusters client key through the config-store. This makes sense to me?